manifests/stage-rdma-device-plugin/0060_rdma-shared-dev-plugin-ds.yaml

```diff
@@ -45,6 +45,14 @@ spec:
       - key: nvidia.com/gpu
         operator: Exists
         effect: NoSchedule
+{{if .DeployInitContainer}}
+      initContainers:
+        - name: ofed-driver-validation
+          image: {{ .CrSpec.Repository }}/{{ .CrSpec.Image }}:{{ .CrSpec.Version }}
+          imagePullPolicy: IfNotPresent
+          command: [ 'sh', '-c' ]
+          args: [ "until lsmod | grep mlx5_core; do echo waiting for OFED drivers to be loaded; sleep 30; done" ]
```

Comment:
I am not sure this validation is good enough? did you talk to OFED team?
don't we need to check the rdma ulp modules are loaded?

Reply:
We do the same checks in the OFED container [1]. If we need more checks, we need fix them everywhere.

[1] https://github.com/Mellanox/network-operator/blob/e73e49b99ff9d2f0eb92ab95143a9b4890f96d07/manifests/stage-ofed-driver/0050_ofed-driver-ds.yaml#L90

Reply:
I see, let first talk to ofed and see what they recommend

Reply:
by the way why we need the sleep?

Reply:
Since there are neither initialDelaySeconds nor periodSecond for init containers, we need to have some timeout between checks

Reply:
I tested init container with `test -e /run/mellanox/drivers/.driver-ready` script few dozens time. it looks less stable rather than kernel module check. e.g.: after I stopped MOFED pod I've got the following state:

```
 root@node-1:~# lsmod | grep mlx
 root@node-1:~# ls /run/mellanox/drivers/.driver-ready
 /run/mellanox/drivers/.driver-ready
```

It seems to be a MOFED issue which should be troubleshooted and fixed separately. I prefer to check for kernel module now

Reply:
ok let keep it for now and open issue in github to track it.